cloudant-client/src/main/java/com/cloudant/client/api/model/Membership.java

```diff
@@ -43,15 +43,15 @@
     }
 
     public String toString() {
-        String ret = "all_nodes: ";
+        StringBuilder ret = new StringBuilder("all_nodes: ");
         for (String s : all_nodes) {
-            ret += s + ",";
+            ret.append(s + ",");

```

Comment:
I'm not sure if this makes any difference in terms of efficiency and temporary strings, but how about
```java
ret.append(s);
ret.append(",");
```